### DISCUSSION TITLE
How to Chagne Height Datalist  ?

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6174] [Help] [answered]
![Height](https://user-images.githubusercontent.com/29665710/147869187-29ec2b97-c566-47cf-bfef-c848a2bc5e50.jpg)

**Comment by dfaruque** [comment, marked as answer]
put the following line in the constructor of yourEditor.ts
`            container.css({ minHeight: 230 });`

**Reply by ra-bedo11** [reply]
Thanks so much

Good
